docs/release-notes/highlights-2.12.0.asciidoc

```diff
@@ -1,6 +1,13 @@
 [[release-highlights-2.12.0]]
 == 2.12.0 release highlights
 
+[float]
+[id="{p}-2120-known-issues"]
+=== Known issues
+
+- When upgrading Elasticsearch to 8.13.0, the operator will be stalled in the upgrade process due to a reconciler error where the Elasticsearch client failed
+to request the desired nodes API. There is no workaround to resolve this issue. You must update the operator to the next patch release.
```

Comment:
During the upgrade of Elasticsearch to version 8.13.0, the operator may encounter a stall in the process due to a reconciler error, wherein the Elasticsearch client fails to request the desired nodes API. Regrettably, there is no workaround available to resolve this issue. The only solution is to update the operator to the subsequent patch release.

Reply:
Sounds better thanks. I took it without regrettably.